corehq/apps/reports/v2/reports/explore_case_data.py

```diff
@@ -109,6 +117,18 @@ def get_data_response(self, endpoint):
 
         expressions = []
         for column_context in endpoint.report_context.get('columns', []):
+            if column_context.get('sort'):
```

Comment:
The front-end only supports sorting by one column at a time, is that right?

Reply:
yes